ldelf/ldelf.ld.S

```diff
@@ -82,5 +81,5 @@ SECTIONS {
 	.data : { *(.data .data.* .gnu.linkonce.d.*) }
 	.bss : { *(.bss .bss.* .gnu.linkonce.b.* COMMON) }
 
-	/DISCARD/ : { *(.interp) }
+	/DISCARD/ : { *(.interp) *(.eh_frame .eh_frame.*) }
```

Comment:
I think this should be part of a separate commit applied first: "ldelf: discard .eh_frame .eh_frame* sections".
Then the second one would change only the compiler flag explaining the sections are already discarded in all the users of core-platform-cflags, i.e. the TEE core and ldelf.